packages/styles/src/sass/components/text-input/_index.scss

```diff
@@ -0,0 +1,233 @@
+.cbp-input__label {
+  color: var(--cbp-color-text-darkest);
+  display: inline-block;
+  font-size: var(--cbp-font-size-body);
+  font-style: italic;
+  font-weight: var(--cbp-font-weight-bold);
+  margin-bottom: var(--cbp-space-2x);
+}
+
+.cbp-input__description {
+  color: var(--cbp-color-text-darkest);
+  font-size: var(--cbp-font-size-subhead);
+  font-style: italic;
+  font-weight: var(--cbp-font-weight-medium);
+  margin-bottom: var(--cbp-space-2x);
+
+  &--error {
+    color: var(--cbp-color-danger-base);
+  }
+}
+
+.cbp-input {
+  background-color: var(--cbp-color-white);
+  border-color: var(--cbp-color-interactive-neutral-base);
+  border-style: solid;
+  border-width: var(--cbp-border-size-md);
+  border-radius: var(--cbp-border-radius-soft);
+  color: var(--cbp-color-text-darkest);
+  font-size: var(--cbp-font-size-body);
+  font-weight: var(--cbp-font-weight-regular);
+  letter-spacing: var(--cbp-letter-spacing-regular);
+  min-height: 2.25rem;
+  padding: var(--cbp-space-2x);
+  width: 100%;
```

Comment:
Can we set this only if :not([size]) and for textareas :not([cols]) ?

Reply:
done